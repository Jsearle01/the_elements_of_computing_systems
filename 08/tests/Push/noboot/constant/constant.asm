@32767
D=A
@SP
M=M+1
A=M-1
M=D
@SP
M=M+1
A=M-1
M=0
@SP
M=M+1
A=M-1
M=1
@666
D=A
@SP
M=M+1
A=M-1
M=D
