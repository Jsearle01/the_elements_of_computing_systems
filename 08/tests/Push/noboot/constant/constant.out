| RAM[0] | RAM[1] | RAM[2] |RAM[256]|RAM[257]|RAM[258]|RAM[259]|RAM[260]|RAM[261]|
|    260 |    101 |    102 |  32767 |      0 |      1 |    666 |      0 |      0 |
